conf/application.conf

```diff
@@ -1,11 +1,22 @@
-play.server.http.port = disabled
-play.server.https.port = 9443
+play {
+  server {
+    http.address = localhost
+    http.address = ${?TRANSPORT_HTTP_BIND_ADDRESS}
+    https.address = localhost
+    https.address = ${?TRANSPORT_HTTPS_BIND_ADDRESS}
+    # Ports are hardcoded and the values match the values in the `play-grpc.yml` kubernetes
+    # descriptor. We could use ENV_VAR overwrites to make this more flexible too.
+    http.port = 9000
+    https.port = 9443
+  }
+}
+
```

Comment:
Allow hardcoded values in dev/test and override points in deployment. Also: open both 9000 and 9433 simultaneously!